curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/task-23.md

```diff
@@ -0,0 +1,27 @@
+---
+id: 657b5ee9cec5d47ffbf67d4f
+title: Task 23
+challengeType: 22
+dashedName: task-23
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!-- (audio) Sophie: Hey, Tom! I saw you taking lots of pics with your cell phone outside the building. Are you into photography? -->
+
+# --description--
+
+The word `outside` refers to the area or space that is not inside a particular building or room. In the context of the dialogue, Sophie mentions that she saw Tom taking pictures in the area that is not inside the building.
```

Comment:
This looks like a possible duplicate of Task 20

<img width="825" alt="Screenshot 2023-12-16 at 10 23 02 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/8e390a8e-d91f-4f4f-bbda-0d4edd67d457">